eta/core/datasets.py

```diff
@@ -673,13 +677,16 @@ def has_data_with_name(self, data_path):
         data_file = os.path.basename(data_path)
         return data_file in self._data_to_labels_map
 
-    def add_file(self, data_path, labels_path, move_files=False,
+    def add_file(self, data_path, labels_path=None, new_data_path=None,
+                 new_labels_path=None, move_files=False,
                  error_on_duplicates=False):
         '''Adds a single data file and its labels file to this dataset.
 
         Args:
             data_path: path to data file to be added
             labels_path: path to corresponding labels file to be added
+            new_data_path: filename for the data file to be renamed to
+            new_labels_path: filename for the labels file to be renamed to
```

Comment:
new_data_path: optional filename for the data file to be renamed to
new_labels_path: optional filename for the labels file to be renamed to